backend/src/idl/policy.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package policy;
+
+option java_multiple_files = true;
+
+message PolicyDetails {
+    WorkSchedule policy = 1;
```

Comment:
are fields by default required or optional in proto? Regardless, lets mention which fields are required in this file and which are not. I am sure, not all of them can be optional